### PR TITLE
Update Operator-SDK version

### PR DESCRIPTION
**Pull request by halim-lee**
Updates Operator-sdk version to 1.24.0 and dependencies

### CODE REVIEW

**Comment by halim-lee** [comment]
Tests passed: ci-pipelinerun #787

**Comment by halim-lee** [comment]
Includes changes in https://github.com/WASdev/websphere-liberty-operator/pull/398

**Comment by halim-lee** [comment]
Tests passed: Pipeline run 838

File: bundle/manifests/ibm-websphere-liberty.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1037,7 +1108,4 @@ spec:
   maturity: stable
   provider:
     name: IBM
-  relatedImages:
```
Comment (leochr) [inline bundle/manifests/ibm-websphere-liberty.clusterserviceversion.yaml:1040]: This should be retained for Airgap (offline install) support. It comes from the [config/manifests/bases/ibm-websphere-liberty.clusterserviceversion.yaml](https://github.com/WASdev/websphere-liberty-operator/pull/389/files#diff-d16ec231c96d3d888f885fe8d75d7351ea35a69f4e480302f1f4945f6adec35b).

File: bundle/manifests/wlo-controller-manager_v1_serviceaccount.yaml [hunk 2]
```diff
@@ -1,9 +0,0 @@
-apiVersion: v1
```
Comment (leochr) [inline bundle/manifests/wlo-controller-manager_v1_serviceaccount.yaml:1]: I believe we had to add this in the first place to change the service account name in CSV (wlo-controller-manager as opposed to controller-manager). I am guessing that the prefix `wlo-` is automatically added by Operator-SDK to Service Account?

File: scripts/create-fyre-stack.sh [hunk 3]
```diff
@@ -146,7 +146,7 @@ parse_args() {
   fi
 
   if [[ -z "${VM_SIZE}" ]]; then
-    readonly VM_SIZE="s"
+    readonly VM_SIZE="m"
```
Comment (leochr) [inline scripts/create-fyre-stack.sh:149]: Is this to resolve the kind test issue?

**Review by leochr — COMMENTED** [review]
@halim-lee Thanks for the PR. Looks good! Added a couple of questions.

File: scripts/create-fyre-stack.sh (hunk 3 above)
Comment (halim-lee) [reply scripts/create-fyre-stack.sh:149]: Yes, small size VM generates a lot of timeout failures for Kind e2e test.

**Review by leochr — APPROVED** [review]
@halim-lee Thanks for the updates!!

**Pull request merged**